tests/geocoder/test_data/.gitignore

```diff
@@ -0,0 +1 @@
+!*.json
```

Comment:
Does that get around the .gitignore in the parent directory ignoring *.json? (I'm relatively new to Git.)

Reply:
Yes! You can `add --force` any file explicitly, even if you told git to ignore it through gitignore.
https://www.atlassian.com/git/tutorials/saving-changes/gitignore
You can also just make an exception for the specific file(s) that you don't want to ignore in the main .gitignore file.

At this point, though, since the data files are already tracked, it is sufficient to simply delete this .gitignore file and whatever is currently tracked by git will stay tracked.